credsweeper/ml_model/ml_validator.py

```diff
@@ -2,30 +2,44 @@
 import os
 import pathlib
 import string
-from typing import List, Tuple, Union
+from typing import List, Tuple, Union, Any
 
+from credsweeper.common.constants import ThresholdPreset, DEFAULT_ENCODING
+from credsweeper.credentials import Candidate
+from credsweeper.logger.logger import logging
+
+ML_VALIDATOR_IMPORT_ERROR = "Start importing"
 try:
     import numpy as np
     import tensorflow as tf
     from tensorflow.keras import models
     from tensorflow.python.keras.backend import set_session
     from tensorflow.python.keras.preprocessing.sequence import pad_sequences
     from tensorflow.python.keras.utils.np_utils import to_categorical
+    from credsweeper.ml_model import features
+
+    ML_VALIDATOR_IMPORT_ERROR = None
 except ModuleNotFoundError as e:
-    raise ModuleNotFoundError("The ML Validation function cannot be used without additional ML packages.\n"
-                              f"{e.msg}\n"
-                              "Run `pip install credsweeper[ml]` to fix it.")
+    class np:
```

Comment:
Rather than declaring type to avoid errors in type hints, it's possible to use "forward references"
https://www.python.org/dev/peps/pep-0484/#forward-references
This should still resolve all references related to types, but should not error on execution 

For example by changing
```python
def encode(cls, line, char_to_index) -> np.ndarray:
```
to 
```python
def encode(cls, line, char_to_index) -> "np.ndarray":
```

Reply:
The sources are under instrumental by atheris - it does not know about --ml_validation

Reply:
If atheris can work with `"np.ndarray"` format, it's probably a more clean way to solve this issue, rather than create a new dummy class

For example IDEs (PyCharm) do properly understand this syntax:

<img src="https://user-images.githubusercontent.com/43581724/154240068-e1d697b0-6061-45e8-981a-0ac7ef4fe383.png" width="550"/>

<img src="https://user-images.githubusercontent.com/43581724/154240065-7f698338-8ef5-4556-8b85-0ca32ab559ec.png" width="550"/>

Reply:
OK